test/e2e/protractor.on-prepare.js

```diff
@@ -0,0 +1,11 @@
+const protractorUtils = require('@angular/bazel/protractor-utils');
```

Comment:
since this repo is the canonical example, could you add more explanatory comments about when this file executes and how it relates to the protractor rule and the code under test?

Reply:
👍